src/class-wpml-elementor-urls.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class WPML_Elementor_URLs implements IWPML_Action {
+
+	/** @var SitePress  */
+	private $sitepress;
```

Comment:
You need to declare `$element_factory` and `$domains` (`$sitepress` is not use AFAICS).

Reply:
Removed.